charts/karmada/values.yaml

```diff
@@ -102,6 +102,9 @@ preInstallJob:
 staticResourceJob:
   tolerations: []
   nodeSelector: {}
+  ## Set a TTL for the static-resource Job, the Job will be automatically cleaned up after this time.
+  ## This only works on Kubernetes version 1.23 or higher.
+  ttlSecondsAfterFinished: 10
```

Comment:
What's the impact if we don't clean the job?

Reply:
The finished Pod holds extra resources, e.g., IP. Since this Job is useless, we need to release those resources.

Reply:
Do you mean the Pod IP, right? Yeah, I agree that we should try to clean up temporary resources after the installation.
I'm asking because I want to evaluate the benefit we can get from this. Thanks for the clarification.